examples/crowdfunding/index.rsh

```diff
@@ -0,0 +1,57 @@
+'reach 0.1';
+
+const Song = Tuple(Bytes(64), Bytes(12), UInt) // Title, id, Price
```

Comment:
You should use an NFT-style approach where they just have numbers and the details are published off-chain, but you commit a hash of the information